scripts/alpine-postgresql-setup-and-start.sh

```diff
@@ -21,7 +26,9 @@ echo "Enabling connections on all available IP interfaces..."
 echo "listen_addresses='*'" >> "$PGDATA/postgresql.conf"
 
 echo "Starting PostgreSQL..."
-pg_ctl start -D "$PGDATA"
+pg_ctl start --pgdata="$PGDATA" --log="/tmp/postgresql-server-start.log"
+sleep 1
+cat "/tmp/postgresql-server-start.log"
```

Comment:
Write logs to the `tmp` file instead of `stdout` to avoid "hanging" when running 'pg_ctl start', also described in this [issue](https://github.com/sindresorhus/execa/issues/1191), and after waiting for 1 sec, print the log file content for debugging.